src/main/java/org/apache/cassandra/distributed/shared/AbstractBuilder.java

```diff
@@ -64,6 +64,16 @@
     private int datadirCount = 3;
     private final List<Rack> racks = new ArrayList<>();
     private boolean finalised;
+    private int tokenCount = getDefaultTokenCount();
+    private boolean allowVnodes = true;
+
+    protected int getDefaultTokenCount() {
+        String key = "cassandra.dtest.num_tokens";
+        String value = System.getProperty(key);
+        if (value == null)
+            value = System.getenv(key.replace(".", "_").toUpperCase());
+        return value == null ? 1 : Integer.parseInt(value);
+    }
```

Comment:
https://the-asf.slack.com/archives/CK23JSY2K/p1643747361709199

mck asked we rely on system properties via ant rather than environment variable, but figure I will keep environment variable anyways as its useful outside of Apache CI

Reply:
mck and I settled on just adding `-Dtest.jvm.args` to ant to pass through w/e we want, and that CI will use the system property method; leaving environment variable as its easier in some cases (its what I actually use)